crates/ir/src/graphviz/function.rs

```diff
@@ -0,0 +1,136 @@
+use std::iter;
+
+use dot2::{label::Text, GraphWalk, Id, Labeller, Style};
+
+use crate::{value::DisplayArgValues, Block, ControlFlowGraph, Function, InsnData};
+
+use super::block::BlockNode;
+
+pub(super) struct FunctionGraph<'a>(pub &'a Function)
+where
+    Self: 'a;
+
+impl<'a> FunctionGraph<'a> {
+    pub(super) fn blocks(&self) -> Vec<BlockNode<'a>> {
+        let Self(func) = *self;
+        let dummy_block = Block(u32::MAX);
```

Comment:
`u32::MAX` is often used in the `graphviz` module, it'd be nice to make it const like
`const DUMMY_BLOCK: Block = Block(u32::MAX);`